### PR TITLE
chore(generate): update routing files & add --routing to ng new

### PR DESCRIPTION
**Pull request by Brocco**

### CODE REVIEW

File: packages/angular-cli/utilities/find-parent-module.ts [hunk 1]
```diff
@@ -11,10 +11,9 @@ export default function findParentModule(project: any, currentDir: string): stri
   let pathToCheck = path.join(sourceRoot, currentDir);
 
   while (pathToCheck.length >= sourceRoot.length) {
-    // let files: string[] = fs.readdirSync(pathToCheck);
-
-    // files = files.filter(file => file.indexOf('.module.ts') > 0);
+    // TODO: refactor to not be based upon file name
     const files = fs.readdirSync(pathToCheck)
+      .filter(fileName => !fileName.endsWith('routing.module.ts'))
```
Comment (filipesilva) [inline packages/angular-cli/utilities/find-parent-module.ts]: There's a scenario where someone has a single module called `routing` and we'll never find it, but tbh I don't think we should special-case it this far... when we refactor we'll handle it.

Comment (Brocco) [inline packages/angular-cli/utilities/find-parent-module.ts]: This is going to go away, the file name string matching is temporary.

**Comment by oravecz** [comment]
`ng serve` currently results in errors in the console after running `ng new --routing myapp`

```
Error: Cannot match any routes: ''
    at resolvePromise (zone.js:429)
    at zone.js:406
    at ZoneDelegate.invoke (zone.js:203)
    at Object.onInvoke (ng_zone_impl.js:43)
    at ZoneDelegate.invoke (zone.js:202)
    at Zone.run (zone.js:96)
    at zone.js:462
    at ZoneDelegate.invokeTask (zone.js:236)
    at Object.onInvokeTask (ng_zone_impl.js:34)
    at ZoneDelegate.invokeTask (zone.js:235)
```

**Comment by Brocco** [comment]
@oravecz include the options after the command + arguements. So in your example, run" 
```
ng new myapp --routing
```

**Pull request merged**